pkg/kubelet/cm/topologymanager/fake_topology_manager.go

```diff
@@ -29,30 +31,30 @@ type fakeManager struct {
 }
 
 // NewFakeManager returns an instance of FakeManager
-func NewFakeManager() Manager {
-	klog.InfoS("NewFakeManager")
+func NewFakeManager(ctx context.Context) Manager {
```

Comment:
We are missing corresponding updates to the `container_manager_windows.go` as it refers to topologymanager.NewFakeManager(). Context needs to be passed to it.